drivers/SmartThings/zwave-switch/src/multi-metering-switch/init.lua

```diff
@@ -0,0 +1,140 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local st_device = require "st.device"
+local capabilities = require "st.capabilities"
+--- @type st.zwave.CommandClass
+local cc = require "st.zwave.CommandClass"
+--- @type st.zwave.CommandClass.Meter
+local Meter = (require "st.zwave.CommandClass.Meter")({version = 3})
+--- @type st.zwave.CommandClass.Basic
+local Basic = (require "st.zwave.CommandClass.Basic")({ version = 1, strict = true })
+--- @type st.zwave.CommandClass.SwitchBinary
+local SwitchBinary = (require "st.zwave.CommandClass.SwitchBinary")({version = 2, strict = true })
+
+local PARENT_ENDPOINT = 1
+
+local MULTI_METERING_SWITCH_FINGERPRINTS = {
+  {mfr = 0x0086, prod = 0x0003, model = 0x0084}, -- Aeotec Nano Switch 1
+  {mfr = 0x0086, prod = 0x0103, model = 0x0084}, -- Aeotec Nano Switch 1
+  {mfr = 0x0086, prod = 0x0203, model = 0x0084}, -- AU Aeotec Nano Switch 1
+  {mfr = 0x027A, prod = 0xA000, model = 0xA004}, -- Zooz ZEN Power Strip 1
+  {mfr = 0x015F, prod = 0x3102, model = 0x0201}, -- WYFY Touch 1-button Switch
+  {mfr = 0x015F, prod = 0x3102, model = 0x0202}, -- WYFY Touch 2-button Switch
+  {mfr = 0x015F, prod = 0x3102, model = 0x0204}, -- WYFY Touch 4-button Switch
+  {mfr = 0x015F, prod = 0x3111, model = 0x5102}, -- WYFY Touch 1-button Switch
+  {mfr = 0x015F, prod = 0x3121, model = 0x5102}, -- WYFY Touch 2-button Switch
+  {mfr = 0x015F, prod = 0x3141, model = 0x5102}, -- WYFY Touch 4-button Switch
+}
+
+local function can_handle_multi_metering_switch(opts, driver, device, ...)
+  for _, fingerprint in ipairs(MULTI_METERING_SWITCH_FINGERPRINTS) do
+    if device:id_match(fingerprint.mfr, fingerprint.prod, fingerprint.model) then
+      return true
+    end
+  end
+  return false
+end
+
+local function get_profile(device)
+  if device:get_manufacturer() == 0x015F then -- only WYFY's have switch-binary
+    return "switch-binary"
+  else
+    return "metering-switch"
+  end
+end
+
+local function is_zooz_power_strip(device)
+  if device:get_manufacturer() == 0x027A and device:get_model() == 0xA004 then
+    return true
+  end
+end
```

Comment:
> @KKlimczukS I have similar thoughts about it. Maybe sub-sub-drivers should be created instead of current checks.

@dsobierajsk this PR previously contained a configuration file that returned number of child devices and a child device profile. I'm wondering here if we should use it again and rely on device.zwave_endpoints[] if the configuration file doesn't return anything.